docs - remove Sentinel from enterprise features list (#18176)
hashicorp/consul: website/content/docs/dynamic-app-config/kv.mdx

you avoid the use of `*`, `?`, `'`, and `%` because they can cause issues when
using the API and in shell scripts.

## Using Sentinel to apply policies for Consul KV 

<EnterpriseAlert>

This feature requires
HashiCorp Cloud Platform (HCP) or self-managed Consul Enterprise.

</EnterpriseAlert>

> [Sentinel](/consul/docs/enterprise/sentinel) could also be leverage as a Policy-as-code framework for defining advanced key-value storage access control policies. Sentinel policies extend the ACL system in Consul beyond static "read", "write",

Reviewer: ```suggestion
You can also use [Sentinel](/consul/docs/enterprise/sentinel) as a Policy-as-code framework for defining advanced key-value storage access control policies. Sentinel policies extend the ACL system in Consul beyond static "read", "write",
```

active voice

and "deny" policies to support full conditional logic and integration with
external systems. Reference the [Sentinel documentation](https://docs.hashicorp.com/sentinel/concepts) for high-level Sentinel concepts.

> To get started with Sentinel in Consul,
> [read the general documentation](https://docs.hashicorp.com/sentinel/consul) or
> [Consul documentation](/consul/docs/agent/sentinel).

Reviewer: ```suggestion
To get started with Sentinel in Consul,
refer to the [Sentinel documentation](https://docs.hashicorp.com/sentinel/consul) or
[Consul documentation](/consul/docs/agent/sentinel).
```
